app.js

```diff
@@ -0,0 +1,109 @@
+'use strict';
+
+const express = require('express');
+const app = express();
+const path = require('path');
+const logger = require('./lib/logger');
```

Comment:
please continue to use it as is at the moment i.e. as an integral part of the app. It's a low priority, but also needs some more thought. We actually use "churchill" and "winston" for logging, our `./lib/logger` is just the configuration of the logger.